### PR TITLE
Step 3 framework

### PR DESCRIPTION
**Pull request by lekshmimallika-aot**
*Issue #:* /bcgov/entity#3027

*Description of changes:* 

Step 3 data structure and framework updates. 
 - From what I understand, share class and share series has the same set of attributes. Share class c 
   an have an array of share series as child elements based on an attribute value.

Step 3 is enabled now  so that everyone can check the flow. We can leave it as enabled or make it disabled till step 3 is fully ready.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: src/interfaces/stepper-interfaces/CreateShareStructure/create-share-structure-interface.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+export interface ShareStructureIF {
+    valid: boolean;
+    shareClasses : ShareClassIF[];
+}
+
+export interface ShareClassIF {
+    id: number| null;
+    displayOrder: number; // To support move up and down functionality
```
Comment (severinbeauvais) [inline src/interfaces/stepper-interfaces/CreateShareStructure/create-share-structure-interface.ts]: Or do we want to simply reorder the array of shares?

Comment (severinbeauvais) [reply src/interfaces/stepper-interfaces/CreateShareStructure/create-share-structure-interface.ts]: Then again, when we restore a draft, we can't guarantee the order of an array, so leave this in.

**Comment by severinbeauvais** [comment]
> * From what I understand, share class and share series has the same set of attributes. Share class c
>   an have an array of share series as child elements based on an attribute value.

Will you update the data store for share series in this ticket?

> Step 3 is enabled now so that everyone can check the flow. We can leave it as enabled or make it disabled till step 3 is fully ready.

I think you can leave it enabled.

File: src/views/CreateShareStructure.vue [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+      </header>
+    </section>
+</template>
+
+<script lang="ts">
+// Libraries
+import { Component, Vue } from 'vue-property-decorator'
+
+@Component({})
+export default class CreateShareStructure extends Vue {}
```
Comment (cameron-freshworks) [inline src/views/CreateShareStructure.vue]: one thing i've noticed from trying to resume drafts is we need a way to re-validate after reloading a draft. I'm not sure the best solution on this, but i've fiddle around with the validating methods in a `mounted` hook in the frameworks. Just something to think about, maybe you have a better idea :)

**Pull request merged**